### ISSUE TITLE
No notifications of new Direct Messages

### ISSUE THREAD

**Issue by ValdikSS** [baedert/corebird#743]
It seems that the client receives DM from server and marks it as read on server, but doesn't show notification (blue dot) in DM tab. No notification is also seen in twitter web interface as twitter thinks it's already read. The only way to know you have new DM is to open DM tab from time to time.

**Comment by Vistaus** [comment]
Can't reproduce, it works fine for me. What OS are you using and more importantly: which Corebird version?

**Comment by ValdikSS** [comment]
Fedora 25, Corebird 1.5.
I have a lot of issues with Corebird, only one of which is discussed (#709).
You assumed that it could be account issue.
Just FYI, I tried to clean Corebird configuration multiple times, it doesn't help.

**Comment by Vistaus** [comment]
Wowow, easy there. I never assumed anything, I just said that it works fine for me but that doesn't mean it's an account issue. It could be anything, really.

**Comment by ValdikSS** [comment]
I'm talking about issue #709 
>Still can't reproduce. Seems like it's an issue with your account or the Twitter server.

**Comment by Vistaus** [comment]
I was talking to IBBoard in that thread, not to you, and I also mentioned that it could be a Twitter server issue there. Could be the same here. But who knows, it could be anything. All I know is that I can't reproduce it, so at least it's not a general issue but it is an issue for some people. Which makes it even harder to debug.

**Comment by ValdikSS** [comment]
How can I help you? Is there a way to provide debug information from my account?

**Comment by baedert** [comment]
Eh, you really can't mark a DM as read on the Twitter server.

But yes, we only download new DMs once you go to the DM threads page. One bug I just found by looking at this is that we always only download newer ones from the current highest cached ID, meaning if you get a DM while corebird is running and the cache is old, going to the DMs page won't download DMs in between those two. i.e. you...

 - Receive >=1 DM while corebird is closed
 - Open corebird
 - get a DM
 - go check the DM

and now the ones you received while corebird was closed are missing and will never be downloaded.
